tests/test_client.py

```diff
@@ -68,77 +73,189 @@ def test_upload_zip(self):
         self.assertEqual("updated-label55", item55["label"])
         self.session.delete(self.ref_entity, 'ref55')
 
-    def test_add_kwargs(self):
+    def test_update_one_error(self):
         try:
-            self.session.delete(self.ref_entity, 'ref55')
+            self.session.update_one(self.ref_entity, 'ref555', 'label', 'updated-label555');
         except Exception as e:
-            print(str(e))
+            message = e.args[0]
+            expected = "404 Client Error:  for url: http://localhost:8080/api/v1/org_molgenis_test_python_TypeTestRef/ref555/label: Unknown entity with 'value' 'ref555' of type 'TypeTestRef'."
+            self.assertEqual(expected, message)
+
+    def test_add_kwargs(self):
+        self._try_delete(self.ref_entity, ['ref55'])
         self.assertEqual('ref55', self.session.add(self.ref_entity, value="ref55", label="label55"))
-        item55 = self.session.get(self.ref_entity, q=[{"field": "value", "operator": "EQUALS", "value": "ref55"}])[0]
-        self.assertEqual({"value": "ref55", "label": "label55", "href": "/api/v1/" + self.ref_entity + "/ref55"},
+        item55 = self.session.get(self.ref_entity, q="value==ref55")[0]
+        self.assertEqual({"value": "ref55", "label": "label55", "_href": "/api/v2/" + self.ref_entity + "/ref55"},
                          item55)
         self.session.delete(self.ref_entity, 'ref55')
 
     def test_add_merge_dict_kwargs(self):
-        try:
-            self.session.delete(self.ref_entity, 'ref55')
-        except Exception as e:
-            print(str(e))
+        self._try_delete(self.ref_entity, ['ref55'])
         self.assertEqual('ref55', self.session.add(self.ref_entity, {"value": "ref55"}, label="label55"))
-        item55 = self.session.get(self.ref_entity, q=[{"field": "value", "operator": "EQUALS", "value": "ref55"}])[0]
-        self.assertEqual({"value": "ref55", "label": "label55", "href": "/api/v1/" + self.ref_entity + "/ref55"},
+        item55 = self.session.get(self.ref_entity, q="value==ref55")[0]
+        self.assertEqual({"value": "ref55", "label": "label55", "_href": "/api/v2/" + self.ref_entity + "/ref55"},
                          item55)
         self.session.delete(self.ref_entity, 'ref55')
 
+    def test_get(self):
+        data = self.session.get(self.ref_entity)
+        expected = [{'_href': '/api/v2/org_molgenis_test_python_TypeTestRef/ref1', 'value': 'ref1', 'label': 'label1'},
+                    {'_href': '/api/v2/org_molgenis_test_python_TypeTestRef/ref2', 'value': 'ref2', 'label': 'label2'},
+                    {'_href': '/api/v2/org_molgenis_test_python_TypeTestRef/ref3', 'value': 'ref3', 'label': 'label3'},
+                    {'_href': '/api/v2/org_molgenis_test_python_TypeTestRef/ref4', 'value': 'ref4', 'label': 'label4'},
+                    {'_href': '/api/v2/org_molgenis_test_python_TypeTestRef/ref5', 'value': 'ref5', 'label': 'label5'}]
+        self.assertEqual(data, expected)
+
+    def test_get_raw(self):
```

Comment:
This test fails on my machine

Reply:
It should be fixed now.

Reply:
I can't run the tests at all now :( 

```Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/helpers/pycharm/_jb_unittest_runner.py", line 35, in <module>
    main(argv=args, module=None, testRunner=unittestpy.TeamcityTestRunner, buffer=not JB_DISABLE_BUFFERING)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/main.py", line 100, in __init__
    self.parseArgs(argv)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/main.py", line 147, in parseArgs
    self.createTests()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/main.py", line 159, in createTests
    self.module)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/loader.py", line 220, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/loader.py", line 220, in <listcomp>
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/Users/tommy/git/molgenis-py-client/tests/test_client.py", line 6, in <module>
    class TestStringMethods(unittest.TestCase):
  File "/Users/tommy/git/molgenis-py-client/tests/test_client.py", line 18, in TestStringMethods
    session.login('admin', 'admin')
  File "/Users/tommy/git/molgenis-py-client/molgenis/client.py", line 58, in login
    self._token = response.json()['token']
TypeError: string indices must be integers```

Reply:
I'm confused about the failing tests on your end. For me everything works perfect: 
![Screen Shot 2019-05-20 at 08 42 33](https://user-images.githubusercontent.com/10075439/58001248-41178e00-7adb-11e9-934c-6a1a66e6e4cb.png)
Any suggestions?

It seems like you don't get the response back the way the client expects. Is your molgenis running on localhost:8080?

Reply:
My bad: I was running a broken version of MOLGENIS